cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/deploy/pipeline/AppDeploymentInfo.java

```diff
@@ -88,10 +89,10 @@ public NamespaceId getNamespaceId() {
   }
 
   /**
-   * Returns the class name of the {@link Application}.
+   * Returns the {@link io.cdap.cdap.api.artifact.ApplicationClass} associated with this {@link Application}.
```

Comment:
no need to use fully qualified name.

Reply:
Missed this, will fix in a new checkin.

Reply:
Created PR https://github.com/cdapio/cdap/pull/12833